apps/odyssey-stats/webpack.config.js

```diff
@@ -31,12 +31,12 @@ const cachePath = path.resolve( '.cache', extraPath );
 const excludedPackages = [
 	/^calypso\/components\/inline-support-link$/,
 	/^calypso\/components\/web-preview.*$/,
-	/^calypso\/blocks\/upsell-nudge.*$/,
 	/^calypso\/my-sites\/stats\/mini-carousel.*$/,
 	/^calypso\/blocks\/jetpack-backup-creds-banner.*$/,
 	/^calypso\/components\/data\/query-keyring-connections$/,
 	/^calypso\/components\/data\/query-jetpack-modules$/,
 	/^calypso\/components\/data\/query-site-keyrings$/,
+	/^calypso\/components\/data\/query-preferences$/,
```

Comment:
Stop calls to preferences which doesn't exist in Jetpack, and won't likely be supported because it's bond to wpcom users.